### ISSUE TITLE
Regression in 1.1.2: android.support.v4.app.NotificationCompat.WearableExtender does not seem to be migrated

### ISSUE THREAD

**Issue by kryops** [dpa99c/cordova-plugin-androidx-adapter#16]
# Bug report

**CHECKLIST**
- [x] I have read the [issue reporting guidelines](README#reporting-issues)

- [x] I confirm this is a suspected bug or issue that will affect other users

- [ ] I have reproduced the issue using the example projector provided the necessary information to reproduce the issue.

- [x] I have read the documentation thoroughly and it does not help solve my issue.

- [x] I have checked that no similar issues (open or closed) already exist.

**Current behavior:**

After upgrading to 1.1.2, I get a build error in my project. 1.1.1 works fine.

**Steps to reproduce:**

`cordova run android`

**Environment information**

- Cordova CLI version 
	- `cordova -v` 10.0.0
- Cordova platform version
	- `cordova platform ls` 9.0.0
- Plugins & versions installed in project (including this plugin)
    - `cordova plugin ls`

```
cordova-background-geolocation-lt 3.9.2 "BackgroundGeolocation"
cordova-plugin-actionsheet 2.3.3 "ActionSheet"
cordova-plugin-androidx-adapter 1.1.1 "cordova-plugin-androidx-adapter"
cordova-plugin-background-fetch 6.1.1 "CDVBackgroundFetch"
cordova-plugin-camera 5.0.0 "Camera"
cordova-plugin-dbcopy 2.1.2 "sqlDB"
cordova-plugin-device 2.0.3 "Device"
cordova-plugin-dialogs 2.0.2 "Notification"
cordova-plugin-file-opener2 2.1.4 "File Opener2"
cordova-plugin-file-transfer 2.0.0-dev "File Transfer"
cordova-plugin-file 6.0.2 "File"
cordova-plugin-geolocation 4.0.2 "Geolocation"
cordova-plugin-ionic-webview 5.0.0 "cordova-plugin-ionic-webview"
cordova-plugin-splashscreen 6.0.0 "Splashscreen"
cordova-plugin-statusbar 2.4.4-dev "StatusBar"
cordova-plugin-whitelist 1.3.4 "Whitelist"
cordova-sqlite-evplus-ext-free 0.4.1 "Cordova sqlite storage - free enterprise version [evplus version] with Android performance/memory improvements and TBD other extra features for PhoneGap Build"
cordova-support-google-services 1.3.2 "cordova-support-google-services"
cordova-windows-capability-private-network 1.0.1 "Windows Add internetClientServer and privateNetworkClientServer capabilities"
phonegap-plugin-barcodescanner 8.1.0 "BarcodeScanner"
phonegap-plugin-multidex 1.0.0 "Multidex"
phonegap-plugin-push 2.3.0 "PushPlugin"
uk.co.workingedge.phonegap.plugin.launchnavigator 5.0.5 "Launch Navigator"
```

- Dev machine OS and version, e.g.
    - OSX
        - `sw_vers` ProductName:    Mac OS X, ProductVersion: 10.15.7, BuildVersion:   19H2

_Android build issue:_	
- Node JS version
    - `node -v` 12.16.3
- Gradle version
	- `ls platforms/android/.gradle` 6.5.1
- Target Android SDK version
	- `android:targetSdkVersion` in `AndroidManifest.xml` 29
- Android SDK details
	- `sdkmanager --list | sed -e '/Available Packages/q'`

<details>
<summary>Android SDK details</summary>

```
Installed packages:
  Path                                                                              | Version | Description                                     | Location                                                                          
  -------                                                                           | ------- | -------                                         | -------                                                                           
  build-tools;28.0.3                                                                | 28.0.3  | Android SDK Build-Tools 28.0.3                  | build-tools/28.0.3/                                                               
  build-tools;29.0.3                                                                | 29.0.3  | Android SDK Build-Tools 29.0.3                  | build-tools/29.0.3/                                                               
  build-tools;30.0.2                                                                | 30.0.2  | Android SDK Build-Tools 30.0.2                  | build-tools/30.0.2/                                                               
  cmdline-tools;latest                                                              | 2.1     | Android SDK Command-line Tools (latest)         | cmdline-tools/latest/                                                             
  emulator                                                                          | 30.1.5  | Android Emulator                                | emulator/                                                                         
  extras;android;m2repository                                                       | 47.0.0  | Android Support Repository                      | extras/android/m2repository/                                                      
  extras;google;m2repository                                                        | 58      | Google Repository                               | extras/google/m2repository/                                                       
  extras;intel;Hardware_Accelerated_Execution_Manager                               | 7.5.1   | Intel x86 Emulator Accelerator (HAXM installer) | extras/intel/Hardware_Accelerated_Execution_Manager/                              
  extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2 | 1       | Solver for ConstraintLayout 1.0.2               | extras/m2repository/com/android/support/constraint/constraint-layout-solver/1.0.2/
  extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2        | 1       | ConstraintLayout for Android 1.0.2              | extras/m2repository/com/android/support/constraint/constraint-layout/1.0.2/       
  patcher;v4                                                                        | 1       | SDK Patch Applier v4                            | patcher/v4/                                                                       
  platform-tools                                                                    | 30.0.4  | Android SDK Platform-Tools                      | platform-tools/                                                                   
  platforms;android-26                                                              | 2       | Android SDK Platform 26                         | platforms/android-26/                                                             
  platforms;android-28                                                              | 6       | Android SDK Platform 28                         | platforms/android-28/                                                             
  platforms;android-29                                                              | 5       | Android SDK Platform 29                         | platforms/android-29/                                                             
  platforms;android-30                                                              | 3       | Android SDK Platform 30                         | platforms/android-30/                                                             
  sources;android-29                                                                | 1       | Sources for Android 29                          | sources/android-29/                                                               
  system-images;android-29;google_apis_playstore;x86                                | 8       | Google Play Intel x86 Atom System Image         | system-images/android-29/google_apis_playstore/x86/                               
  system-images;android-30;google_apis;x86                                          | 8       | Google APIs Intel x86 Atom System Image         | system-images/android-30/google_apis/x86/                                         
  system-images;android-30;google_apis_playstore;x86                                | 8       | Google Play Intel x86 Atom System Image         | system-images/android-30/google_apis_playstore/x86/                               
  tools                                                                             | 26.1.1  | Android SDK Tools                               | tools/                                                                            

```

</details>

**Console output**

```
> Task :app:compileDebugJavaWithJavac FAILED
/.../platforms/android/app/src/main/java/com/adobe/phonegap/push/FCMService.java:27: error: package android.support.v4.app.NotificationCompat does not exist
import android.support.v4.app.NotificationCompat.WearableExtender;
                                                ^
/.../platforms/android/app/src/main/java/com/adobe/phonegap/push/FCMService.java:604: error: cannot find symbol
        mBuilder.extend(new WearableExtender().addActions(wActions));
                            ^
  symbol:   class WearableExtender
  location: class FCMService
```

**Comment by dpa99c** [comment]
Ah this is caused by the fix for #15 because `android.support.v4.app.NotificationCompat.WearableExtender` is not explicitly mapped in the [class mappings](https://developer.android.com/jetpack/androidx/migrate/class-mappings) - the `android.support.v4.app.NotificationCompat` package is mapped instead.
I will adjust the regex so it allows a package name to be mapped but not a partial class name (which caused the problem highlighted in #15).

**Comment by dpa99c** [comment]
Fix published in v1.1.3 - please give that a try and let me know if you still have issues

**Comment by kryops** [comment]
Looks good, thanks a lot for the quick fix!
